### PR TITLE
Fix tuple table replacement

### PR DESCRIPTION
**Pull request by RobinPapke**

### CODE REVIEW

File: pypika/tests/test_query.py [hunk 1]
```diff
@@ -120,6 +115,20 @@ def test_replace_orderby_table(self):
...
+        query = (
+            Query.from_(self.table_a)
+            .select(self.table_a.cost, self.table_a.revenue)
+            .where(Tuple(self.table_a.cost, self.table_a.revenue) == Tuple(1, 2))
+        )
+
+        query = query.replace_table(self.table_a, self.table_b)
+
+        self.assertEqual(
+            "SELECT " '"cost","revenue" ' 'FROM "b" ' 'WHERE ("cost","revenue")=(1,2)',
```
Comment (mikeengland) [inline pypika/tests/test_query.py:128]: Is this SQL string correct? It has multiple opening/closing single quotes.

File: pypika/tests/test_query.py [hunk 2]
```diff
@@ -120,6 +115,20 @@ def test_replace_orderby_table(self):
             'SELECT "customer" ' 'FROM "b" ' 'ORDER BY "customer"', str(query)
         )
 
+    def test_replace_tuple_table(self):
+        query = (
+            Query.from_(self.table_a)
+            .select(self.table_a.cost, self.table_a.revenue)
+            .where(Tuple(self.table_a.cost, self.table_a.revenue) == Tuple(1, 2))
```
Comment (twheys) [inline pypika/tests/test_query.py:122]: You shouldn't need to use the Tuple class directly. You can just use a python tuple. Pypika wraps python tuples with it's own tuple internally.

File: pypika/tests/test_query.py (hunk 1 above)
Comment (RobinPapke) [reply pypika/tests/test_query.py]: Fixed the formatting of the sql strings.

File: pypika/tests/test_query.py (hunk 2 above)
Comment (RobinPapke) [reply pypika/tests/test_query.py]: I've updated it to use a python tuple instead.

**Pull request merged**